1_fetch.R

```diff
@@ -58,7 +58,8 @@ p1_targets_list <- list(
       get_nwis_sites(drb_huc8s,pcodes_select,site_tp_select,stat_cd_select)
     },
     deployment = 'main',
-    cue = tar_cue(mode = NWIS_cue)
+    cue = tar_cue(mode = NWIS_cue),
+    repository = 'aws'
```

Comment:
Instead of defining "aws" as the repository for each of these NWIS_cue targets, can you add a variable called NWIS_repository that gets defined under NWIS_cue?